jobs/scheduled/enqueue_reminders.rb

```diff
@@ -12,7 +12,7 @@ def execute(_args)
     private
 
     def skip_enqueue?
-      SiteSetting.remind_assigns_frequency.nil? || !SiteSetting.assign_enabled?
+      SiteSetting.remind_assigns_frequency.nil? || !SiteSetting.assign_enabled? || !SiteSetting.assign_allowed_on_groups.present?
```

Comment:
```suggestion
      SiteSetting.remind_assigns_frequency.nil? || !SiteSetting.assign_enabled? || SiteSetting.assign_allowed_on_groups.blank?
```

I find `blank?` reads easier than `!present?`